.github/workflows/prchecks.yml

```diff
@@ -103,11 +108,6 @@ jobs:
         run: lintr::lint_package()
         shell: Rscript {0}
 
-      - name: Goodpractice
-        run: |
-          goodpractice::gp()
-        shell: Rscript {0}
-
```

Comment:
Can we replace this with the new workflow instructions from `{manydata}` (or was it `{migraph}`)?